extension-requests/script.js

```diff
@@ -0,0 +1,221 @@
+const container = document.querySelector('.container');
+const extensionRequestsContainer = document.querySelector(
+  '.extension-requests',
+);
+
+const errorHeading = document.querySelector('h2#error');
+const modalParent = document.querySelector('.extension-requests-modal-parent');
+const closeModal = document.querySelectorAll('#close-modal');
```

Comment:
The `querySelectorAll()` method should only be used when you need to select multiple elements. In this case, you only need to select one element, so you should use the `querySelector()` method instead.

Reply:
yes, and that is exactly why I have used `querySelectorAll`